### ISSUE TITLE
[Suggestion] Character List Folders

### ISSUE THREAD

**Issue by DivertedCircle** [Orcpub/orcpub#52]
**Is your feature request related to a problem? Please describe.**
The Character List page does not allow sorting, instead displaying all characters alphabetically. Adding folders and allowing sorting will make for a shorter list

**Describe the solution you'd like**
Allow custom sorting including folder functionality.

**Comment by MrZwij** [comment]
I agree with comments in #119 that parties essentially already serve as folders. But they would be more useful if they were:
- Collapsible
- Sortable

**Comment by kamander** [comment]
Yes, please allow collapsing/expanding of parties.

**Comment by LAProbert** [comment]
The party function is not really a suitable way to organise characters.
We really need folders and sorting on the character tab.

Character tab
  Folder name
    Character 1
    Character 2
  Folder name 2
    Character 3
  Character 4
  Folder name 3
    Character 5

And also sorting/filter options as well e.g. by level, by class, by highest class (for multiclass)

**Comment by 4sfaloth** [comment]
> I agree with comments in #119 that parties essentially already serve as folders. But they would be more useful if they were:
> 
> * Collapsible
> * Sortable

Thumbs for this! I came here to suggest this exact same feature. I DM for several different groups and keep their characters organized in parties. I also use the character builder for some major NPCs and/or enemies which means I now have maybe around 30+ character or so, making it somewhat hard to navigate. I manage by simply using CTRL+F for the party I'd looking for, but it really would be great to have parties collapsible and sortable
